hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/AzureBlobFileSystem.java

```diff
@@ -476,6 +476,20 @@ public FileStatus getFileStatus(final Path f) throws IOException {
     }
   }
 
+  /**
+   * Acquire a lease on an ABFS file for a specified duration. This requires the file to exist.
```

Comment:
little architecture question. Would this be better in the Store than the FS? I don't know, and it is higher level than the Rest API, isn't it? Which implies this is the right place.